packages/@okta/vuepress-theme-prose/assets/css/abstracts/_colors.scss

```diff
@@ -8,6 +8,34 @@ $colors: (
     'base': #E22866,
     'active':#dd2864,
   ),
```

Comment:
Are there remaining usages of the old `'brand'` and `'cerise'` variables?

Reply:
I still use cerise for the form errors, the rest I'll remove